website/content/docs/installing/no-gen-resources.mdx

```diff
@@ -0,0 +1,466 @@
+  -password "foofoofoo" \
+  -auth-method-id <auth_method_id_from_last_step"
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_account" "myuser" {
+  name           = "myuser"
+  description    = "User account for my user"
+  type           = "password"
+  login_name     = "myuser" 
+  password       = "foofoofoo"
+  auth_method_id = boundary_auth_method.password.id
+}
+```
+
+</Tab>
+</Tabs>
+
+#### Create a User
+
+Create a user and associate the user with the login account created in the previous step.
+This user will also be the principal in the role we create in the following step.
+
+<Tabs>
+<Tab heading="CLI">
+
+```bash
+$ boundary users create -scope-id <org_scope_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -name "myuser" \
+  -description "My user!"
+
+$ boundary users add-accounts -id <myuser_user_id> -account <myuser_account_id>
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_user" "myuser" {
+  name        = "myuser"
+  description = "My user!"
+  account_ids = [boundary_account.myuser.id]
+  scope_id    = boundary_scope.org.id
+}
+```
+
+</Tab>
+</Tabs>
+
+
+#### Create Roles to Manage Scopes
+
+The follow example creates 4 roles:
+1. To allow anonymouns (unauthenticated) users the ability to list scopes and auth methods in the global scope.
+1. To allow anonymouns (unauthenticated) users the ability to list scopes and auth methods in the orgnaization scope.
+1. To allow `myuser` user administration grants at the org scope.
+1. To allow `myuser` user administration grants at the project scope.
+
+1. Create anonymous listing role for global scope
+
+<Tabs>
+<Tab heading="CLI">
+
+Assumes recovery key export from above steps is still set:
+
+```bash
+# Create global anonymous listing role
+$ boundary roles create -name 'global_anon_listing' \
+  -recovery-config /tmp/recovery.hcl \
+  -scope-id 'global'
+
+$ boundary add-grants -id <global_anon_listing_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -grant 'id=*;type=auth-method;actions=list,authenticate'
+
+$ boundary add-grants -id <global_anon_listing_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -grant 'type=scope;actions=list'
+
+$ boundary add-grants -id <global_anon_listing_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -grant 'id=*;actions=read,change-password'
+
+$ boundary add-principals -id <global_anon_listing_id> \
+  -recovery-config /tmp/recovery.hcl \
+  -principal 'u_anon' 
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+```hcl
+resource "boundary_role" "global_anon_listing" {
+  scope_id = "global"
+  grant_strings = [
+    "id=*;type=auth-method;actions=list,authenticate",
+    "type=scope;actions=list",
+    "id=*;actions=read,change-password"
+  ]
+  principal_ids = ["u_anon"]
+}
+```
+
+</Tab>
+</Tabs>
+
+2. Create anonymous listing role for org scope
+
+<Tabs>
+<Tab heading="CLI">
+
+Assumes recovery key export from above steps is still set:
+
+```bash
+$ boundary roles create -name 'global_anon_listing' \
```

Comment:
Did you mean for global_anon_listing here and below to be something like "org_anon_listing"?